### PR TITLE
Test fails on Mac InstructorFeedbackEditPageUiTest line 133 #4273

### LINKED ISSUE #4273: Test fails on InstructorFeedbackEditPageUiTest due to datepicker element
Timed out after waiting 15 seconds for presence of element, it seems to be a Mac issue but I'll see how I can improve it to pass consistently across machines.

### PR DESCRIPTION
**Pull request by junhaoyap**
Fixes #4273 

@LowWeiLin please help take a look :)

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 1]
```diff
@@ -529,9 +529,7 @@ public boolean verifyEndDatesBeforeTodayAreDisabled() {
             String year = Integer.toString(cal.get(Calendar.YEAR));
 
             // go to previous month
-            while (!(getDatepickerMonth().equals(month) && getDatepickerYear().equals(year))) {
-                browser.driver.findElement(By.className("ui-datepicker-prev")).click();
-            }
+            navigateToRightMonthAndYear(month, year);
```
Comment (LowWeiLin) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: How do we ensure that the javascript executor has finished running before proceeding?

Comment (LowWeiLin) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: I got a failure on the test, probably due to that.

**Comment by LowWeiLin** [comment]
The problem seems to happen when the browser is not in focus. Maybe we can try to trigger some focus event? I'm not exactly sure if it will work though.

**Comment by junhaoyap** [comment]
I'll look into how we can ensure the JavaScript is executed before we continue running the other stuff. However, before this change the test will fail if the browser is not in focus, there is no way to forcefully push the browser to the front unfortunately :\

**Comment by LowWeiLin** [comment]
Yes, but can we send an event using javascript to make the date picker think the browser is in focus and appear?

**Comment by junhaoyap** [comment]
I don't think so leh, the last time I worked on a similar issue, I researched on Selenium and whether there was any way we could bring the browser to focus / forefront and turns out there wasn't as far as I know, I can look into it again, after the hardcoded admin milestone ba?

**Comment by LowWeiLin** [comment]
I'm not saying to actually focus the window, but to trigger the focus event using JS, so that the date-picker thinks there is focus.

**Comment by junhaoyap** [comment]
Can try, that will help us figure out whether the problem is the window not being in the foreground or whether the focus on the date picker ui element can help with it showing, get back to you soon!

**Pull request closed without merging**